### PR TITLE
Fat 1466 tc 345404

### PR DESCRIPTION
**Pull request by dmitriyfolio**

### CODE REVIEW

**Review by KyryloBrener — APPROVED** [review]

File: cypress/integration/orders/orders.unreceivePiece.spec.js [hunk 1]
```diff
@@ -7,6 +7,7 @@ import TopMenu from '../../support/fragments/topMenu';
 import Helper from '../../support/fragments/finance/financeHelper';
 import InventorySearch from '../../support/fragments/inventory/inventorySearch';
 import InventoryInstance from '../../support/fragments/inventory/inventoryInstance';
+import interactorsTools from '../../support/utils/interactorsTools';
```
Comment (MaksimTerekhov) [inline cypress/integration/orders/orders.unreceivePiece.spec.js:10]: Let's use same case for first letter

File: cypress/integration/finance/funds/funds.deleteBudgetWithTransfer.spec.js [hunk 2]
```diff
@@ -1,16 +1,17 @@
 import Funds from '../../../support/fragments/finance/funds/funds';
 import TopMenu from '../../../support/fragments/topMenu';
-import { getCurrentFiscalYearCode } from '../../../support/utils/dateTools';
-import testType from '../../../support/dictionary/testTypes';
+import DateTools from '../../../support/utils/dateTools';
+import TestType from '../../../support/dictionary/testTypes';
 import FinanceHelp from '../../../support/fragments/finance/financeHelper';
 import newFund from '../../../support/fragments/finance/funds/newFund';
```
Comment (MaksimTerekhov) [inline cypress/integration/finance/funds/funds.deleteBudgetWithTransfer.spec.js:6]: Let's use same case for first letter

File: cypress/integration/quickmark/holding-quickmark.spec.js [hunk 3]
```diff
@@ -8,6 +8,8 @@ import QuickMarcEditor from '../../support/fragments/quickMarcEditor';
 import InventoryViewSource from '../../support/fragments/inventory/inventoryViewSource';
 import testTypes from '../../support/dictionary/testTypes';
```
Comment (MaksimTerekhov) [inline cypress/integration/quickmark/holding-quickmark.spec.js:9]: Let's use same case for first letter

**Review by MaksimTerekhov — APPROVED** [review]

File: cypress/integration/finance/funds/funds.deleteBudgetWithTransfer.spec.js (hunk 2 above)
Comment (dmitriyfolio) [reply cypress/integration/finance/funds/funds.deleteBudgetWithTransfer.spec.js:6]: I will change with in new pr

File: cypress/integration/orders/orders.unreceivePiece.spec.js (hunk 1 above)
Comment (dmitriyfolio) [reply cypress/integration/orders/orders.unreceivePiece.spec.js:10]: I will change with in new pr

**Pull request merged**